docs/3_Scope.md

```diff
@@ -0,0 +1,8 @@
+# Scope
+This document outlines the various use cases that are in and out of scope for this specification.
+
+## Use Cases In Scope
+This version of the specification covers the following categories of the Local-Retail
+
+
+## Use Cases Out of Scope
```

Comment:
Mention Out of Scope as : 
1. Repair of Item
2. Buying & Selling at Bulk
3. Buying & Selling of any Financial service